v2/session/common-customizations/sessions/new-session.mdx

```diff
@@ -222,6 +222,89 @@ export default async function superTokens(req: SessionRequest, res: any) {
 }
 ```
 
+</TabItem>
+<TabItem value="nextjs-app">
+
+```tsx
+import { serialize } from "cookie";
+import { NextResponse, NextRequest } from "next/server";
+import SuperTokens from "supertokens-node";
+import { CollectingResponse, PreParsedRequest } from "supertokens-node/framework/custom";
+import { HTTPMethod } from "supertokens-node/types";
+import { createNewSession } from "supertokens-node/recipe/session";
+// @ts-ignore
+import { backendConfig } from "@/app/config/backend";
+
+SuperTokens.init(backendConfig());
+
+export async function POST(req: NextRequest) {
```

Comment:
missed using withPreParsedRequestResponse here? Please double check your code.

Reply:
Fixed.

Reply:
Not fixed. There is no change in this file since i last reviewed.

Reply:
Not sure how it got removed from the commit. Have done the changes now.